client/my-sites/domains/domain-management/contacts-privacy/index.jsx

```diff
@@ -37,12 +37,15 @@ class ContactsPrivacy extends React.PureComponent {
 			return <DomainMainPlaceholder goBack={ this.goToEdit } />;
 		}
 
-		const { translate } = this.props;
+		const { translate, whois } = this.props;
 		const domain = getSelectedDomain( this.props );
 		const { hasPrivacyProtection, privateDomain, privacyAvailable, currentUserCanManage } = domain;
 		const contactInformation = privateDomain
-			? findPrivacyServiceWhois( this.props.whois.data )
-			: findRegistrantWhois( this.props.whois.data );
+			? findPrivacyServiceWhois( whois.data )
+			: findRegistrantWhois( whois.data );
+
+		// eslint-disable-next-line
+		console.log( contactInformation );
```

Comment:
I think this line needs to be removed ;)

Reply:
Thanks @klimeryk!! Er, I left that in there just as a test... you know... 😙

Reply:
Admit it - you were checking if I really did read the code or just threw out random nitpicks 😁